CleverTapSDK/CTLocalDataStore.m

```diff
@@ -624,11 +609,8 @@ - (BOOL)isEventLoggedFirstTime:(NSString*)eventName {
             return NO;
         }
     }
-    if (!self.dbHelper) {
-        self.dbHelper = [CTEventDatabase sharedInstanceWithConfig:self.config];
-    }
     // TODO: Add normalized name here
```

Comment:
Remove TODO comment if code is updated at other place also

Reply:
lets keep them until we get a complete flow working,